install-vmod/install-vmod

```diff
@@ -16,7 +16,7 @@
 set -e
 
 check_bin() {
-	if ! command $@ &>/dev/null; then
+	if ! command "$@" &>/dev/null; then
```

Comment:
Now I'm wondering what the `&` is about.

Did you mean `>/dev/null 2>&1` instead?

Reply:
hum, looks like it's a bashism that means exactly that, fixed.